src/CustomerProfile.js

```diff
@@ -0,0 +1,12 @@
+import React, { useEffect } from "react";
+
+function CustomerProfile(props) {
```

Comment:
Use as much as arrow function in new JS development as possible to avoid JS lang, quirks. `https://www.freecodecamp.org/news/learn-es6-the-dope-way-part-ii-arrow-functions-and-the-this-keyword-381ac7a32881/`